packages/api-kintone/src/andpadPayments/updateAndpadRecords.ts

```diff
@@ -0,0 +1,27 @@
+import { RecordID, Revision, UpdateKey } from '@kintone/rest-api-client/lib/src/client/types';
+import { updateRecords } from 'api-kintone';
+import { RecordType, appId } from './config';
+
+
+export type UpdateAndpadPayments = {
+  id: RecordID
+  record?: Partial<RecordType> | undefined
+  revision?: Revision | undefined
+} | {
+  updateKey: UpdateKey
+  record?: Partial<RecordType> | undefined
+  revision?: Revision | undefined
+};
```

Comment:
```suggestion
export type UpdateAndpadPayments = Parameters<typeof updateRecords>[0]['records'];
```

updateRecordsの型の引数も利用出来ます。配列になりますので、もし、当`suggestion`を適用するなら、`updateAndpadRecords`の引数の型から`[]`を省く必要があると思います。

Reply:
ご指摘ありがとうございます。
修正しましたので、ご確認をお願いします。